src/bfcli/parser.y

```diff
@@ -133,7 +135,7 @@ chains          : chain
                 }
                 ;
 
-chain           : CHAIN hook raw_hook_opts POLICY verdict rules
+chain           : CHAIN hook raw_hook_opts POLICY policy rules
```

Comment:
Keep the `verdict` rule here, and if `policy == CONTINUE` then trigger `bf_parse_err()`.  This way we reuse the existing code, and we provide a contextual error (e.g. "CONTINUE not supported for chains").